components/badges/badges.css

```diff
@@ -0,0 +1,37 @@
+.relative{
+    position: relative;
+    z-index: 0;
+}
+.absolute{
+    position:absolute;
+    z-index: 1;
+}
+.dot{
+    width:15px;
+    border-radius: 50%;
+    height:15px;
+    right: 0px;
+    bottom: 0px;
+    border: 2px solid var(--PURE-WHITE);
+}
+.online{
+    background-color: var(--SUCCESS-GREEN);
+}
+.offline{
+    background-color: var(--495057);
+}
+.badge{
+    font-size: 2rem;
+    width:30px;
```

Comment:
just a suggestion, since `rem` & `px` both are used. You can make it consistent & responsive by using `rem` unit across.

Reply:
Yeah for sure I will do that . Thanks for reviewing it